app/config/sources.inc.php

```diff
@@ -276,6 +280,11 @@
         'priority'          => 1,
         'supported_locales' => $mozillaorg,
     ],
+    'firefox/quantum.lang' => [
+        'deadline'          => '2017-09-26',
+        'priority'          => 1,
+        'supported_locales' => ['de'],
```

Comment:
Question: should this be open to other languages at some point?

Reply:
I asked and I was told explicitly this is for de only.  There will be a "global" quantum copy ready on Oct 3.